src/containers/about/constants/index.js

```diff
@@ -84,7 +84,7 @@ const MATTHEW :TeamMemberType = {
 
 const REKA :TeamMemberType = {
   NAME: 'Réka Halmai',
-  ROLE: 'Data Scientist Intern',
+  ROLE: 'Data Scientist',
```

Comment:
Please use the `DATA_SCIENTIST` constant.

Reply:
ah ok! Thanks :) Done